src/Libraries/Microsoft.Extensions.AI.AzureAIInference/README.md

```diff
@@ -281,6 +281,10 @@ app.MapPost("/chat", async (IChatClient client, string message) =>
 app.Run();
 ```
 
+## Documentation
+
+Refer to the [Microsoft.Extensions.AI libraries documentation](https://learn.microsoft.com/dotnet/ai/microsoft-extensions-ai) for more information and API usage examples.
```

Comment:
I'm not sure if we have anything in the docs about this package. Should we? So not sure if it makes sense to link there.

Reply:
We do not need docs specifically about M.E.AI.AzureAIInference. I think it's ok to link to the general M.E.AI docs.